contracts/wawaka/CMakeLists.txt

```diff
@@ -12,10 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required (VERSION 3.5 FATAL_ERROR)
+CMAKE_MINIMUM_REQUIRED (VERSION 3.10 FATAL_ERROR)
 
-PROJECT(wawaka-contracts)
+IF (NOT DEFINED ENV{PDO_SOURCE_ROOT})
+  MESSAGE(FATAL_ERROR "PDO_SOURCE_ROOT not defined")
+ENDIF()
+SET(PDO_SOURCE_ROOT $ENV{PDO_SOURCE_ROOT})
```

Comment:
well... you have a circular dependency then... because we NEED pdo_source_root to find ProjectVariables.cmake. This is the only environment variable we should be checking in the cmake files for the moment. with the "uber" cmake we can pass in the variable from the top. for now, i don't see a way around this.

Reply:
Yeah... sounds like the uber cmake file is really the best solution for this.